cg/constants/demultiplexing.py

```diff
@@ -39,50 +56,90 @@ class RunParametersXMLNodes(StrEnum):
     UNKNOWN_REAGENT_KIT_VERSION: str = "unknown"
 
 
-class SampleSheetHeaderColumnNames(StrEnum):
-    DATA: str = "[Data]"
-    FLOW_CELL_ID: str = "FCID"
-    LANE: str = "Lane"
-    SAMPLE_INTERNAL_ID: str = "Sample_ID"
-    SAMPLE_NAME: str = "SampleName"
-    SAMPLE_PROJECT: str = "Sample_Project"
-    CONTROL: str = "Control"
-
-
-SAMPLE_SHEET_HEADERS = {
-    "bcl2fastq": [
-        SampleSheetHeaderColumnNames.FLOW_CELL_ID,
-        "Lane",
-        "SampleID",
-        "SampleRef",
-        "index",
-        "index2",
-        "SampleName",
-        "Control",
-        "Recipe",
-        "Operator",
-        "Project",
-    ],
-    "dragen": [
-        SampleSheetHeaderColumnNames.FLOW_CELL_ID,
-        "Lane",
-        "Sample_ID",
-        "SampleRef",
-        "index",
-        "index2",
-        "SampleName",
-        "Control",
-        "Recipe",
-        "Operator",
-        "Sample_Project",
-    ],
-}
+class SampleSheetNovaSeq6000Sections:
+    """Class with all necessary constants for building a NovaSeqX sample sheet."""
+
+    class Settings(StrEnum):
+        HEADER: str = "[Settings]"
+        BARCODE_MISMATCH_INDEX1: str = "BarcodeMismatchesIndex1, 0"
+        BARCODE_MISMATCH_INDEX2: str = "BarcodeMismatchesIndex2, 0"
+
+    class Data(Enum):
+        HEADER: str = "[Data]"
+        FLOW_CELL_ID: str = "FCID"
+        LANE: str = "Lane"
+        SAMPLE_INTERNAL_ID_BCL2FASTQ: str = "SampleID"
+        SAMPLE_INTERNAL_ID_BCLCONVERT: str = "Sample_ID"
+        SAMPLE_NAME: str = "SampleName"
+        SAMPLE_PROJECT_BCL2FASTQ: str = "Project"
+        SAMPLE_PROJECT_BCLCONVERT: str = "Sample_Project"
+        CONTROL: str = "Control"
+
+        COLUMN_NAMES: Dict[str, List[str]] = {
+            "bcl2fastq": [
+                FLOW_CELL_ID,
+                LANE,
+                SAMPLE_INTERNAL_ID_BCL2FASTQ,
+                "SampleRef",
+                "index",
+                "index2",
+                SAMPLE_NAME,
+                CONTROL,
+                "Recipe",
+                "Operator",
+                SAMPLE_PROJECT_BCL2FASTQ,
+            ],
+            "dragen": [
+                FLOW_CELL_ID,
+                LANE,
+                SAMPLE_INTERNAL_ID_BCLCONVERT,
+                "SampleRef",
+                "index",
+                "index2",
+                SAMPLE_NAME,
+                CONTROL,
+                "Recipe",
+                "Operator",
+                SAMPLE_PROJECT_BCLCONVERT,
+            ],
+        }
+
+
+class SampleSheetNovaSeqXSections:
```

Comment:
This class is currently usused but it will be used on my next PR where I create different SampleSheet classes